include/gz/sim/Types.hh

```diff
@@ -59,6 +60,10 @@ namespace gz
       /// update state when paused is true.
       // cppcheck-suppress unusedStructMember
       bool paused{true};
+
+      /// \brief The namespace of the NetworkManagerSecondary if running a
+      /// distributed simulation. Empty otherwise.
+      std::string secondaryNamespace;
```

Comment:
hmm, I believe this breaks ABI, but it looks like the abi checker is happy. Will need to investigate.